src/configManager/models/database/sequelize/sequelizeConnector.js

```diff
@@ -40,7 +40,7 @@ async function deleteConfig(key) {
 async function getConfig() {
     const configClient = client.model('config');
     const options = {
-        attributes: { exclude: ['updated_at', 'created_at'] }
+        attributes: { exclude: ['updatedAt', 'createdAt'] }
```

Comment:
in sequlize.js in createClient we define

define: {
            underscored: true
        },

not sure why updated_at is now updatedAt
also what about existing users? they have this table already created with underscore name columns

Reply:
You're right, will read about it later here: https://github.com/sequelize/sequelize/issues/6423#issuecomment-379472035
Seems there was a breaking change in underscore

Reply:
/**
   * Override the name of the createdAt column if a string is provided, or disable it if false. Timestamps
   * must be true. Not affected by underscored setting.
   */
  createdAt?: string | boolean;

Reply:
fixed